### ISSUE TITLE
tag new version with upgrade of DimensionalData

### ISSUE THREAD

**Issue by felixcremer** [rafaqz/Rasters.jl#544]
The last tag of Rasters is from beginning of September.
Do you want to include something specific into a new version or could we tag it right away?

I would like to get the update of DimensionalData tagged so that we could use the new Makie recipes in DD.

**Comment by rafaqz** [comment]
Yeah, I only just got the docs actually working last night, DD Makie had some bugs still.

There are currently a few issues with CI not passing, but I should be able to tag a breaking release when they pass.
